cl/custom_filters/templatetags/pacer.py

```diff
@@ -16,6 +16,6 @@ def price(rd: RECAPDocument) -> str:
         return "0.00"
 
     if rd.page_count:
-        cost = rd.page_count * 0.10
+        cost = int(rd.page_count) * 0.10
         return f"{min(3, cost):.2f}"
```

Comment:
Maybe should use [Decimal](https://docs.python.org/3/library/decimal.html) types here instead.

Reply:
Do you think it will affect it significantly given the function is already truncating?